server/database/queries/updateCart.js

```diff
@@ -0,0 +1,8 @@
+const connection = require('../connection');
+
+const updateCart = (userId, productId, quantity) => connection.query({
+  text: 'update cart set quantity = $1 where user_id = $2 and product_id = $3',
```

Comment:
when the client forgets to send a quantity, then the quantity in the database will change to null,
so please handle this case.